app/View/Label.php

```diff
@@ -39,6 +39,10 @@ public function render(callable $callback = null)
         $assets = $this->data->get('assets');
         $offset = $this->data->get('offset');
         $template = $this->data->get('template');
+        
+        if ($assets->isEmpty()){
+            return redirect()->back();
```

Comment:
Should we be doing a `->with('error', 'Blah blah blah no valid assets')` here as well?

Reply:
Thought about that, but if I added that error message the page looks kinda cluttered as there's already an error message showing that the asset  doesn't exist.